website/components/BlogHeader.tsx

```diff
@@ -0,0 +1,73 @@
+import { Avatar } from "./Avatar";
+import cn from "classnames";
+import CLOUDQUERY_TEAM from "../content/team";
+import { getAllPages } from "nextra/context";
+import type { Author } from "../content/team";
+
+type BlogPosFrontMatter = {
+  title: string;
+  /** security */
+  tag: string;
+  /** 2022/06/06 */
+  date: string;
+  description: string;
+  /** yevgenypats */
+  author?: string;
+
+  /** /images/og-image.png */
+  ogImage?: string;
+};
+
+function Authors({ data }: { data: BlogPosFrontMatter }) {
+  const authorName = data?.author as Author;
+
+  if (!authorName) {
+    return null;
+  }
+
+  const author = CLOUDQUERY_TEAM[authorName];
+
+  return (
+    <div className="w-full border-b border-gray-400 authors border-opacity-20">
+      <div className={cn("flex flex-wrap justify-center py-8 mx-auto gap-7")}>
+        <Avatar {...author} />
+      </div>
+    </div>
+  );
+}
+
+function BlogTitle({ data }: { data: BlogPosFrontMatter }) {
+  const title = data.title;
+
+  if (!title) {
+    return null;
+  }
+
+  return <h1>{title}</h1>;
```

Comment:
I used default style for this title which might be weird a little bit, WDYT?

<img width="876" alt="image" src="https://user-images.githubusercontent.com/21976058/183860680-02cbf668-8885-4ac8-bb16-ed415501ba5e.png">